Xamarin.Forms.Platform.Android/Renderers/TimePickerRenderer.cs

```diff
@@ -92,7 +93,8 @@ void OnClick()
 			TimePicker view = Element;
 			ElementController.SetValueFromRenderer(VisualElement.IsFocusedPropertyKey, true);
 
-			_dialog = new TimePickerDialog(Context, this, view.Time.Hours, view.Time.Minutes, false);
+			bool is24HourFormat = DateFormat.Is24HourFormat(Context);
```

Comment:
I was never a fan of nesting method calls within method calls, for reasons of readability and ease of debugging. I don't have strong feeling here though, it's not my code after all - I can change this tomorrow.